### ISSUE TITLE
How to get NUMBER OF PARAMETERS used?

### ISSUE THREAD

**Issue by Lucky2593** [qfgaohao/pytorch-ssd#155]
Can someone please help in telling me how can I get the number of parameters used in the network.

I am training "mobileNetv1-SSDLite" as it started training from scratch(Training base network also).

Command used to train "MobileNetV1-SSDLite"

python train_ssd.py --datasets D:/DataSets/VOCdevkit/VOC2007 D:/DataSets/VOCdevkit/VOC2012 --validation_dataset D:/DataSets/VOCdevkit/VOC2007 --net mb1-ssd-lite --batch_size 24 --num_epochs 200 --scheduler cosine --lr 0.01 --t_max 200
